database/models/index.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+const Sequelize = require('sequelize');
+const process = require('process');
+const basename = path.basename(__filename);
+const env = process.env.NODE_ENV || 'development';
+const config = require(__dirname + '/../config/config.cjs')[env];
+const db = {};
+
+let sequelize;
+if (config.use_env_variable) {
+  sequelize = new Sequelize(process.env[config.use_env_variable], config);
+} else {
+  sequelize = new Sequelize(config.database, config.username, config.password, config);
```

Comment:
Since we modified the database configuration file located here: `config/config.cjs`, consider also replacing the Sequelize instance with:

`const sequelize = new Sequelize(
'database', 'username', 'password' {host: 'host', dialect: "postgres"});` -- replacing parameters with your database credentials. 
You may also refer to this [link](https://sequelize.org/docs/v6/getting-started/#connecting-to-a-database)